yt_dlp/extractor/viewlift.py

```diff
@@ -27,6 +27,7 @@ class ViewLiftBaseIE(InfoExtractor):
         'snagxtreme': 'snagfilms',
         'theidentitytb': 'tampabay',
         'vayafilm': 'snagfilms',
+        "chorki" : "prothomalo",
```

Comment:
there was a linter error because of the whitespace before `:`. also single quotes
```suggestion
        'chorki': 'prothomalo',
```